### PR TITLE
Combine _LayoutMainPage.cshtml and _Layout.cshtml 

### LINKED ISSUE #1071: Combine _LayoutMainPage.cshtml and _Layout.cshtml
`_LayoutMainPage.cshtml` (used only for `/Home/Index`, the home view of the app) has the same purpose as `_Layout.cshtml` and is more or less a duplication of it with some non-essential differences. Reduce maintenance malarkey by combining them into one layout view.

### PR DESCRIPTION
**Pull request by jonbakerappdev**
Fixes #1071

### CODE REVIEW

**Comment by tonysurma** [comment]
@mheggeseth any chance you can review this? @jonbakerappdev it also looks like it needs a rebase after we accepted several PRs.

Thanks @jonbakerappdev for your contributions and welcome to the project!

**Comment by mheggeseth** [comment]
@tonysurma I'm looking at it

File: AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -44,20 +44,24 @@
...
+                    }
                     <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                     </button>
-                    <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand"><img src="~/images/Web-Logo-small.png" alt="Web Logo" /></a>
+                    <div @(descriptor.ActionName == "Index" && controllerName == "Home" ? " style=display:none;" : "")>
+                        <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand"><img  src="~/images/Web-Logo-small.png" alt="Web Logo" /></a>
+                    </div>
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml]: I don't think this change is necessary and it's a little complicated. I think it's totally fine to show the allReady brand image in the navbar on the landing page. This also has the added benefit of visual consistency in the navbar regardless of what view you are on.

File: AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -69,9 +73,15 @@
                 <!--/.nav-collapse -->
             </div>
         </nav>
+        <div class="row logo-background" @(descriptor.ActionName == "Index" && controllerName == "Home" ?  "" : " style=display:none;" )>
+            <div class="col-md-12">
+                <div class="center"><img src="~/images/Web-Logo.png" alt="Logo image for the project" class="img-responsive center-block" /></div>
+                <h3 class="text-center sub-title">Get connected. Work together. Be prepared.</h3>
+            </div>
+        </div>
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml]: This hero div really should live in `Home/Index.cshtml`. It's OK if that means the layout changes a little. It's more important for the code to be logically clean.

**Comment by mheggeseth** [comment]
The criteria for whether to consider the "About" link active is no longer sufficient. It needs to verify both that the controller name is `Home` and that the action name is `About`. Alternatively, I would advocate the creation of an `About` controller.

image

File: AllReadyApp/Web-App/AllReady/Views/Shared/_LayoutMainPage.cshtml [hunk 3]
```diff
@@ -1,82 +0,0 @@
...
-        <nav class="navbar">
-            <div class="container-fluid">
-                <div class="navbar-header">
-                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                        <span class="icon-bar"></span>
-                        <span class="icon-bar"></span>
-                        <span class="icon-bar"></span>
-                    </button>
-                </div>
-                @await Html.PartialAsync("_Navigation")
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/Views/Shared/_LayoutMainPage.cshtml]: `Shared/_Nagivation.cshtml` is no longer used and can be deleted.

File: AllReadyApp/Web-App/AllReady/wwwroot/css/mainPage.css [hunk 4]
```diff
@@ -1,115 +0,0 @@
...
-.inline-form {
-    display: inline-block;
-}
-
-.featured-campaign {
-    border: 1px solid #d58033;
-    padding: 6px;
-    margin: 10px;
-    background: #e4cdb8;
-}
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/wwwroot/css/mainPage.css]: The `featured-campaign` styles are still used in `Home/Index.cshtml`. They should either move to `site.css` or to a `styles` section within `Home/Index.cshtml`. See `404.cshtml` for an example.

**Comment by mheggeseth** [comment]
That's all for now. I'll review more tonight or after the rebase.

**Comment by jonbakerappdev** [comment]
I'm looking at it!

**Comment by tonysurma** [comment]
thanks @jonbakerappdev, let me know when you are complete and I can merge

**Comment by tonysurma** [comment]
No rush but wanted to checkin @jonbakerappdev and see if this is ready to go. Thanks

**Comment by MisterJames** [comment]
Superseded by PR #1367.

**Pull request closed without merging**